api/src/main/java/cz/muni/fi/pa165/facade/TimelineFacade.java

```diff
@@ -57,6 +58,13 @@
      */
     List<TimelineDTO> findAllBetweenDates(LocalDate from, LocalDate to);
 
+    /**
+     * Fetches events related to timeline
+     * @param timelineId ID of timeline
+     * @return all timeline events
+     */
+    List<EventDTO> findEventsOfTimeline(Long timelineId);
```

Comment:
Isn't there an easier solution how to get events of timeline? Timeline has list of events, I don't see how this method is necessary.